calblueprint/sundayfriends
# Albert/invite admin modal

- created Invite Admin Modal
- emails sending functionality is functional
- ensured dynamic resizing as new invites are added
- added success modal
- form validation is functional but in a different PR

## Screenshots
![Screen Shot 2021-11-18 at 10 59 29 AM](https://user-images.githubusercontent.com/25868533/142480343-c7696f2d-e7af-4e67-8363-d03e1ad2ed34.png)

![Screen Shot 2021-11-18 at 10 59 50 AM](https://user-images.githubusercontent.com/25868533/142480376-d7a1852b-85a9-4484-9e0e-09a103d1b423.png)

![Screen Shot 2021-11-18 at 10 59 59 AM](https://user-images.githubusercontent.com/25868533/142480395-7968925c-8c1b-41fe-987b-df27162bc364.png)

![Screen Shot 2021-11-18 at 11 00 14 AM](https://user-images.githubusercontent.com/25868533/142480439-a63091a0-9bc8-4a95-839c-b44f63cfdbff.png)

## Related PRs
the modal will only look like this after the "fixed a few styling issues" PR is merged in

## Test Plan
1. Add invites and ensure modal dynamically resizes
2. Input 2 invites (name/email) pairs and submit — check email to ensure emails were delivered
3. Ensure success screen appears on submission
4. Try removing and re-adding invites and ensure input is cleared
5. Ensure all buttons work as expected

CC: @gabriel-jt @cindyxzhang @hukellyy
